src/wallet/coin_selection.rs

```diff
@@ -165,6 +186,7 @@ pub trait CoinSelectionAlgorithm<D: Database>: std::fmt::Debug {
         fee_rate: FeeRate,
         amount_needed: u64,
         fee_amount: u64,
+        _drain_output: &TxOut,
```

Comment:
Since this is the trait definition I wouldn't use the `_`, calling the parameter `drain_output`. Also, drain_output needs to be documented

Reply:
I'll change it by `drain_script: &Script` (with docs and no `_`) as @afilini suggested.